samples/testapp/src/commonMain/kotlin/com/android/identity/testapp/issuance/platform.kt

```diff
@@ -0,0 +1,10 @@
+package com.android.identity.testapp.issuance
+
+import com.android.identity.securearea.CreateKeySettings
+import com.android.identity.securearea.KeyInfo
+import com.android.identity.device.DeviceAttestation
+import com.android.identity.securearea.SecureArea
+
+expect fun platformSecureArea(): SecureArea
+
+expect fun platformKeySetting(clientId: String): CreateKeySettings
```

Comment:
Why not just add this to the existing `Platform.kt` file in `com.android.identity.testapp` package?

Reply:
Done